static/js/custom.js

```diff
@@ -87,3 +87,8 @@ function removeClass(el, cl) {
 function addClass(el, cl) {
     if(el) el.className = el.className + " " + cl;
 }
+
+function incrCommentVote(e) {
+    var node = e.parentNode.parentNode.childNodes[0];
```

Comment:
Nice and simple!  (-:

We can use this as a first approximation, but we will want to have a request that returns the two new values from the server, and a js function that writes them into the right places.  perhaps we can even do that without json/ajax, by violating the form paradigm some more?  (-: